pkg/controller/volume/persistentvolume/pv_controller_test.go

```diff
@@ -160,11 +161,15 @@ func TestControllerSync(t *testing.T) {
 				for len(ctrl.claims.ListKeys()) > 0 {
```

Comment:
Opened https://github.com/kubernetes/kubernetes/issues/93662 for further cleanup